static/js/components/LocalizationPlot.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { PropTypes } from "prop-types";
+import CircularProgress from "@material-ui/core/CircularProgress";
+
+import GeoJSONGlobePlot from "./GeoJSONGlobePlot";
+
+import * as localizationActions from "../ducks/localization";
+
+const LocalizationPlot = ({
+  loc,
+  sources,
+  galaxies,
+  instrument,
+  observations,
+  options,
+}) => {
+  const cachedLocalization = useSelector((state) => state.localization);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(
+      localizationActions.fetchLocalization(loc.dateobs, loc.localization_name)
+    );
+  }, [loc, dispatch]);
+
+  const localization =
+    loc.id === cachedLocalization?.id ? cachedLocalization : null;
+
+  if (!localization || !instrument) {
+    return <CircularProgress />;
+  }
+
+  return (
+    <>
+      <GeoJSONGlobePlot
```

Comment:
Is it possible to get rid of `LocalizationPlot` and just put the code from this component into `GeoJSONGlobePlot`?

Reply:
MErged.